packages/flutter_tools/test/ios/mac_test.dart

```diff
@@ -288,7 +288,7 @@ void main() {
         success: false,
         stdout: '''
 Launching lib/main.dart on iPhone in debug mode...
-Signing iOS app for device deployment using developer identity: "iPhone Developer: test@flutter.io (1122334455)"
+Signing iOS app for device deployment using developer identity: "iPhone Developer: test@flutter.dev (1122334455)"
```

Comment:
@kf6gpe, are the MX records set up to do mail correctly to .dev too?  This is just text, so it doesn't really do anything, but the thought just occured to me...